{"uuid":"cf585e4d-ecc3-44c4-bfe9-c47c8df952e7","name":"Scenario Outline: To validate if user getting error message when enter valid python code in operations stack page","children":["1d830be6-5c8e-405a-9a1c-df9e92fdb222"],"befores":[],"afters":[{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065950805,"stop":1709065950806},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065951417,"stop":1709065951418},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065953026,"stop":1709065953027},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065953387,"stop":1709065953388},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065953721,"stop":1709065953722},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065954454,"stop":1709065954455},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065955057,"stop":1709065955058},{"name":"stepDefinitions.ApplicationHooks.tearDown()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065955058,"stop":1709065955603}],"start":1709065942199,"stop":1709065955605}